dev/bots/docs.sh

```diff
@@ -6,6 +6,8 @@ set -e
 # so are we pinned to this old version until that bug is fixed.
 pub global activate dartdoc 0.9.7+1
 
+pub global activate crossdart
```

Comment:
Do we want to lock this to a version, or version range?

A future version of crossdart could break out build.